frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { expect } from 'chai';
+import {
+  ATTACH_TO_DEPLOYMENT,
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NO_ANNOTATIONS,
+  NOOBAA_LABEL,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  OBC_STORAGE_CLASS_EXACT,
+  SECOND,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
```

Comment:
```suggestion
```
Let's divide this test into only two `it` block. 
The first localized to the details page for the OBC we created. And the second one to the list page(where we only attach to the deployment for now, we will add more tests in the future if required).